people_recognition_3d/src/people_recognition_3d/people_recognizer_3d.py

```diff
@@ -31,6 +32,7 @@ def _get_and_wait_for_service(srv_name, srv_class):
     service.wait_for_service()
     return service
 
+
```

Comment:
Too much whitespace, please revert

Reply:
This is fine. Independent function definitions have two blanks lines between them. In case of class methods, its one. Also there needs to be 2 blank lines after imports.